packages/Manager/sample-data/allocation_data.sql

```diff
@@ -0,0 +1,53 @@
+       ('Te HoroGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-34', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-34"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('Te Ore OreGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-35', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-35"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('TeAwaruaoPoriruaSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-36', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-36"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('Upper HuttGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-37', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-37"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('Upper RuamahangaGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-38', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-38"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaikanaeGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-39', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-39"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaikanaeSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-40', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-40"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaingawaGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-41', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-41"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaingawaSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-42', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-42"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WainuiomataSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-43', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-43"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaiohineSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-44', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-44"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaipouaSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-45', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-45"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('Wairarapa CoastSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-46', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-46"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WairarapaGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-47', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-47"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WaitohuGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-48', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-48"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WgnHarbour,HuttGW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-49', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-49"}', '2000-01-01', '2099-01-01', NOW(), NOW()),
+       ('WgnHarbour,HuttSW', 'SAMPLE-DATA-INGEST', 'SAMPLE-SOURCE-50', 'SAMPLE-CONSENT-ID', 'active', true, 10, '{"TEST-METER-50"}', '2000-01-01', '2099-01-01', NOW(), NOW());
```

Comment:
Maybe something for the future - could we generate these based on existing area-ids in the database?